presto-product-tests/src/main/java/io/prestosql/tests/hive/TestHiveStorageFormats.java

```diff
@@ -185,6 +200,37 @@ public void testInsertIntoPartitionedTable(StorageFormat storageFormat)
         query(format("DROP TABLE %s", tableName));
     }
 
+    @Test(dataProvider = "storage_formats_with_null_format", groups = STORAGE_FORMATS)
+    public void testInsertIntoTableWithNullFormat(StorageFormat storageFormat)
```

Comment:
squash this commit with commit which introduces functionality (`Allow null_format for ...`)